.github/actions/run-integration-tests/action.yml

```diff
@@ -14,6 +14,9 @@ inputs:
   kubernetes_architecture:
     description: 'The CPU architecture used to select arch specific config in tests.'
     required: true
+  dnstester_image:
+    description: 'The image used for the dnstester.'
+    required: false
```

Comment:
use `default: 'ghcr.io/inspektor-gadget/dnstester:latest'` here and remove it from L69?